api/app/tests/apps/api/test_private_source_endpoint.py

```diff
@@ -41,27 +38,82 @@ def test_get_list(self):
         self.assertEqual(5, len(data['results']))
         self.assertJsonSchema(self.list_sources_schema, data)
 
-    def test_get_list_excluding_signal_SOURCE_DEFAULT_ANONYMOUS_USER(self):
-        """
-        Bug: SIG-3934
+    def test_get_list_is_active_filter(self):
+        # setup
+        active_sources = SourceFactory.create_batch(3)
```

Comment:
Not blocker, but specifying `is_active=True` while not necessary because of the SourceFactory's behavior makes the test easier to read. You do do this for `can_be_selected` below.